Documentation/faq.md

```diff
@@ -10,6 +16,24 @@
 
 ### Deployment
 
+#### System requirements
+
+Since etcd writes data to disk, SSD is highly recommended. Default storage size limit is 2GB, configurable up to 8GB. And allocate as much heap size to avoid disk swap. At CoreOS, etcd cluster is usually deployed on dedicated CoreOS Container Linux machines, with dual-core processors, 2GB of RAM, and 80GB of SSD *at the very least*. **It highly varies on workloads; please test before production deployment**.
```

Comment:
why is the default size limit to 2GB?  and why it can only be configurable up to 8gb?
what's the RAM recommendation?

`And allocate as much heap size to avoid disk swap.` is that talking about using a lot of ram to ease disk swap?